src/command/marian_train.cpp

```diff
@@ -68,5 +68,5 @@ int main(int argc, char** argv) {
     }
   }
 
-  return 0;
+  return getSigtermFlag() ? 15 : 0;
```

Comment:
Maybe add the comment from the PR description. And why 15 specifically?

Reply:
Just added comment and changed the exit code to 128 + SIGTERM. See link in comment for explanation.